src/js/pages/Service.js

```diff
@@ -64,19 +65,17 @@ class Service extends Component {
         }
       })
       .then(res => {
-        this.setState({ data: res.data.data.servicePage });
+        const data = this.cleanData(res);
```

Comment:
if we cleanData (get it in the right format) before it's thrown in the state, this will prevent the issues we were seeing where the app would bomb trying to clean already cleaned data in the render method. Basically, this replaces the logic in 'shouldComponentUpdate'